Unix/doc/setup-kerberos-omi.md

```diff
@@ -0,0 +1,627 @@
+
+  - DNS - point at the ADC as the dns name server by editing the source /etc/resolv.conf. Optionally add an entry in /etc/hosts.
+    pointing at the ADC to ensure that is available.
+
+    Note:  If you are using Network Manager, just modifying /etc/resolv.conf will not me sufficient in the long run as 
+    Network Manager clobbers /etc/resolv.conf on every reboot. If so, modify the network manager configuration files 
+    using the redhat-network-configuration tool or as needed. 
+    *but ensure you have a fully qualified domain name by default before proceeding*
+
+  - NTP is necessary. It should point at the ADC plus a number of pool servers. Authentication will fail more or less 
+    mysteriously if the time is out of sync by more than five minutes between the kdc, client and host its trying to 
+    communicate with.
+
+  - Kerberos: Add /etc/krb5.conf.  We recommend that you keep a master krb5.conf and just copy it to each machine. The 
+    contents of /etc/krb5.conf are not host sensitive, though they are of course, unique to the domain.  Realmd and 
+    other authconfigure tools depend on at least a minimal /etc/krb5.conf in order to function.
+
+  - Authconfig will setup the configuration files /etc/nsswitch, /etc/smb.conf, /etc/pam.d/*, /etc/sssd/sssd.conf.
+
+  - Realmd will complete the setup.
+
+  - it will be necessary to add an additional service principal to the  machine account after the join is complete.
+
+
+#### hostname 15 char or less and fully qualified
+
+ In Centos and Red Hat you can alter the hostname by editing /etc/sysconfig/network to be the fully qualified domain name.
+ (and reboot).
+
+#### Resolv.conf points to the ADC
+
+ The effect we are looking for is that the world can ping the host by its fully qualified domain name (myhost.contoso.com), and
+ the host can lookup names in the DNS domain via the nameservers, which we expect to be the Active Domain Controller.
+
+ In Red Hat and Centos, resolv.conf is generated by network manager.  If you edit the file, the modifications 
+ will be clobbered on the next boot.
+
+ So, add an entry for DNS1, DNS2, and DOMAIN into /etc/sysconfig/network-script/if-eth0, then reboot if network
+ manager is active. The resuling ifcfg-eth0 file should look like
+
+
+```
+TYPE="Ethernet"
+BOOTPROTO="dhcp"
+DEFROUTE="yes"
+IPV4_FAILURE_FATAL="no"
+IPV6INIT="yes"
+IPV6_AUTOCONF="yes"
+IPV6_DEFROUTE="yes"
+IPV6_FAILURE_FATAL="no"
+IPV6_ADDR_GEN_MODE="stable-privacy"
+NAME="eth0"
+UUID="xxxxxxxxxx"
+DEVICE="eth0"
+ONBOOT="yes"
+PEERDNS="yes"
+PEERROUTES="yes"
+IPV6_PEERDNS="yes"
+IPV6_PEERROUTES="yes"
+IPV6_PRIVACY="no"
+DNS1=10.xxx.xxx.xxx
+DNS2=10.xxx.xxx.xxx
+DNS_DOMAIN=contoso.com
+DNS_SEARCH=contoso.com
+```
+
+ If Network manager is active, it may be necessary to remove the name server entries already in the resolv.conf.
+ If Network Manager is not active, just edit /etc/resolv.conf. Either way, /etc/resolv.conf should end up looking like 
```

Comment:
No, that's not enough. It also depends on the correct "A" record.

I think you should have ONE section that covers DNS in detail (above). Then refer to that here.